content/docs/300-on-brand-menu-api/400-reference/200-get-menus-options.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: GET /v1/menus/options
+excerpt: Returns a tuple, formatted for use with Remote Options for the authenticated API Key.
+---
+
+Returns a tuple, formatted for use with Remote Options for the authenticated API Key.
+
+## Request
+
+Set the `X-API-Key: [API Key]` header.
+
+```bash
+curl --request GET 'https://menu-api.raydiant.com/v1/menus/options' \
+     --header 'X-API-Key: [API Key]'
+```
+
+## Response
+
+```json
+[
+  {
+    "label": "Test Menu (Location Name)",
```

Comment:
Suggestion: Maybe we can use a menu from one our sample apps instead of "Test Menu"? I find using real data is nice when possible.